src/Utilities/Compiler/Extensions/IOperationExtensions.cs

```diff
@@ -801,6 +801,331 @@ public static bool IsSetMethodInvocation(this IPropertyReferenceOperation operat
 
             return parameterOrderedArguments.ToImmutableArray();
         }
+
+        // Copied from roslyn https://github.com/dotnet/roslyn/blob/master/src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/OperationExtensions.cs#L25
```

Comment:
Copied the whole implementation as it can be useful for existing and future analyzers